src/Bundle/Provider/ServiceGridProvider.php

```diff
@@ -47,6 +48,10 @@ public function __construct(
 
     public function get(string $code): Grid
     {
+        if (is_a($code, GridInterface::class, true)) {
+            $code = $code::getName();
+        }
```

Comment:
Am I the only one who seems to have a problem with the flow here? If the code is GridInterface, then code equals to the name of grid? 😂 Let's maybe at least rename it to an identifier 
```suggestion
        if (is_a($identifier, GridInterface::class, true)) {
            $identifier = $identifier::getName();
        }
````

or let's teach grid registry to look for grids by FQCN

Reply:
Good Idea to rename